sympy/solvers/tests/test_solvers.py

```diff
@@ -1557,23 +1547,61 @@ def test_issue_14607():
         assert s[0][var].simplify() == knownsolution[var].simplify()
 
 
-@slow
 def test_lambert_multivariate():
     from sympy.abc import a, x, y
-    from sympy.solvers.bivariate import _filtered_gens, _lambert, _solve_lambert
-
+    from sympy.solvers.bivariate import _filtered_gens, _lambert
     assert _filtered_gens(Poly(x + 1/x + exp(x) + y), x) == set([x, exp(x)])
     assert _lambert(x, x) == []
     assert solve((x**2 - 2*x + 1).subs(x, log(x) + 3*x)) == [LambertW(3*S.Exp1)/3]
     assert solve((x**2 - 2*x + 1).subs(x, (log(x) + 3*x)**2 - 1)) == \
           [LambertW(3*exp(-sqrt(2)))/3, LambertW(3*exp(sqrt(2)))/3]
     assert solve((x**2 - 2*x - 2).subs(x, log(x) + 3*x)) == \
           [LambertW(3*exp(1 - sqrt(3)))/3, LambertW(3*exp(1 + sqrt(3)))/3]
-    assert solve(x*log(x) + 3*x + 1, x) == [exp(-3 + LambertW(-exp(3)))]
     eq = (x*exp(x) - 3).subs(x, x*exp(x))
     assert solve(eq) == [LambertW(3*exp(-LambertW(3)))]
     # coverage test
     raises(NotImplementedError, lambda: solve(x - sin(x)*log(y - x), x))
+    assert solve(x**3 - 3**x, x) == [3, -3*LambertW(-log(3)/3)/log(3)]
+    assert set(solve(3*log(x) - x*log(3))) == set(  # 2.478... and 3
+        [3, -3*LambertW(-log(3)/3)/log(3)])
+    assert solve(LambertW(2*x) - y, x) == [y*exp(y)/2]
+
+
+@XFAIL
+def test_other_lambert():
+    from sympy.abc import x
+    assert solve(3*sin(x) - x*sin(3), x) == [3]
+    a = S(6)/5
+    assert set(solve(x**a - a**x)) == set(
+        [a, -a*LambertW(-log(a)/a)/log(a)])
+
+@slow
```

Comment:
Why `@slow`? Can you `timeit` and paste the output here (for all tests here) so that we can see how slow these tests go.

Reply:
Timing to run ` test_lambert_bivariate()` ----`5.2713117599487305 seconds`.